.github/workflows/polyglot_release.yml

```diff
@@ -2,81 +2,49 @@ name: Release Polyglot Piranha
 on:
   workflow_dispatch:
 jobs:
-  build_ubuntu:
-    runs-on: ubuntu-latest
+  release:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, ubuntu-20.04, macos-latest]
```

Comment:
how about having a target here instead of the OS? because that's what we are really trying to have a matrix for: `linux-x86, linux-arm, macos-arm`.